### PR TITLE
Statistic and Admins user

### PR DESCRIPTION
**Pull request by docuongg**
image

### CODE REVIEW

**Comment by hnam2812** [comment]
LGTM :+1:

**Comment by docuongg** [comment]
notification
Screenshot from 2023-01-12 09-17-34

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -9,4 +9,8 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.permit :sign_in, keys: [:login, :password]
     devise_parameter_sanitizer.permit :account_update, keys: added_attrs
   end
+
+  def set_notifications
+    @notifications = current_user.notifications.all
```
Comment (hnam2812) [inline app/controllers/application_controller.rb]: paginate

File: app/views/notifications/_notifications.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% if user_signed_in? && @notifications.present? %>
+  <div class="container">
+    <h5>Notification</h5>
+    <hr/>
+    <div style="height: 300px; overflow-y: scroll">
+      <%= turbo_stream_from "notification_#{user.id}" %>
+      <%= turbo_frame_tag "notifications_#{user.id}" do %>
+        <% @notifications.each do |notification| %>
+          <%= render notification %>
```
Comment (hnam2812) [inline app/views/notifications/_notifications.html.erb]: <%= render @notifications %>

**Pull request merged**